src/env/robotmk_env_manifest.rs

```diff
@@ -0,0 +1,78 @@
+use crate::command_spec::CommandSpec;
+use anyhow::{Context, anyhow};
+use camino::Utf8Path;
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub struct RobotmkEnvironmentManifest {
+    #[serde(default)]
+    pub post_build_commands: Vec<PostBuildCommand>,
+}
+
+#[derive(Deserialize)]
+pub struct PostBuildCommand {
+    pub name: String,
+    pub command: String,
+}
+
+impl PostBuildCommand {
+    pub fn try_to_command_spec(&self) -> anyhow::Result<Option<CommandSpec>> {
+        let command_words = shlex::split(&self.command).ok_or(anyhow!(
```

Comment:
This is very likely to be a bad idea, I think. All `.yaml` formats I know don't do this for the user. I am sure there is a reason.

Reply:
I luckily managed to convince Simon :)